### ISSUE TITLE
migrate save and up always recreate the db

### ISSUE THREAD

**Issue by Weakky** [prisma/migrate#316]
## Description

Given the following datamodel

```prisma
datasource db {
  provider = "sqlite" // other options are: "mysql" and "sqlite"
  url      = "sqlite:./dev.db"
}

generator client {
  provider = "prisma-client-js"
}

model User {
  id String @id @default(cuid())
}
```

If you run `prisma2 migrate save --experimental`, you'll be prompted to create a new db if it doesn't exist.

Unfortunately, it'll always ask you to recreate it, even when using `prisma2 migrate up --experimental`

Here's the prompt:

```
You are trying to create a migration for Sqlite database /dev.db.
A database with that name doesn't exist at sqlite:./dev.db.
Do you want to create the database?

┌─ Database options ────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                                                                                                                           │
│ ❯ Yes            Create new Sqlite database /dev.db                                                                                                       │
│   No             Don't create the database                                                                                                                │
│                                                                                                                                                           │
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

It looks like `/dev.db` is used to check if the file already exists, instead of `./dev.db`

**Comment by pantharshit00** [comment]
I am unable to reproduce this: 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/22195362/72914680-0bf7c580-3d65-11ea-9b39-07da96b4cf54.png">

Can you please share your files in a Git reposiotory so that I can replicate the exact environment.

**Comment by mikeslade** [comment]
I am seeing similar issues with sqlite.

**prisma/schema.prisma**
```
datasource db {
  provider = "sqlite"
  url      = "sqlite:./dev.db"
}

generator client {
  provider = "prisma-client-js"
}

model Post {
  id        Int     @id @default(autoincrement())
  title     String
  content   String?
  published Boolean @default(false)
  author    User?
}

model User {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  posts Post[]
}
```
Prisma2 version: 2.0.0-preview020.3

Windows 10

Start with prisma.schema file and package.json... nothing else.

run `yarn` or `npm install`

run `npx prisma2 migrate save --experimental`

select Yes at the create new database prompt

- at this point the dev.db file is created in the appropriate location, but it is empty and the process hangs forever. No further errors or anything logged. I cannot seem to find a way around this. If I cancel the process and try again... it still thinks there is no database file and asks to create a new one.

**Comment by pantharshit00** [comment]
I can confirm this now. This doesn't happen when I use `file` protocol but it happens when I use the `sqlite` protocol.

**Comment by divyenduz** [comment]
Internal note: dropping from current sprint because product

**Comment by mikeslade** [comment]
Re-tested under 2.0.0-preview021 on Windows 10:
When using the `file` protocol and no initial db file, the initial run does create the file, but the process still never ends. If you then kill the terminal and run the migrate command again, it does work as expected.
`sqlite` protocol suffers as before

**Comment by pantharshit00** [comment]
@mikeslade Refer to https://github.com/prisma/migrate/issues/342 for the windows issue.

**Comment by albertoperdomo** [comment]
@Jolg42 will repro

**Comment by Jolg42** [comment]
I could reproduce with `url      = "sqlite:./dev.db` and it works as expected with `url      = "file:./dev.db`

It's also recommended that everybody switches to the `file:`.

**Comment by Jolg42** [comment]
It is now fixed in the latest dev release and will be shipped in 2.5.0 🚢
